plugins/output/flight/flight.go

```diff
@@ -0,0 +1,156 @@
+package flight
+
+import (
+	"context"
+	_ "embed"
+	"log"
+	"net"
+
+	"github.com/apache/arrow/go/v9/arrow"
+	"github.com/apache/arrow/go/v9/arrow/array"
+	"github.com/apache/arrow/go/v9/arrow/flight"
+	"github.com/apache/arrow/go/v9/arrow/ipc"
+	"github.com/apache/arrow/go/v9/arrow/memory"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+// DO NOT REMOVE THE NEXT TWO LINES! This is required to embed the sampleConfig data.
+//
+//go:embed sample.conf
+var sampleConfig string
+
+func (*Flight) SampleConfig() string {
+	return sampleConfig
+}
+
+type Flight struct {
+	Location string `toml:"location"`
+	Port     string `toml:"port"`
+	Table    string `toml:"table"`
+
+	timeStamps []arrow.Timestamp
+	values     []float32
+
+	client flight.FlightServiceClient
+	writer flight.Writer
+	schema arrow.Schema
+	ctx    context.Context
+	pool   memory.GoAllocator
+	desc   *flight.FlightDescriptor
+}
+
+// Write should write immediately to the output, and not buffer writes
```

Comment:
Can you give an example of this? I have read through a lot of the other plugins written for telegraf and have yet to see anything like this.

Reply:
Well if it is generally not done in the Go community it may be better to follow their coding style than what we do for Rust. What do you think @CGodiksen and @chrthomsen? For examples from ModelarDB, see, e.g., [`UncompressedDataManager.insert_message()`](https://github.com/ModelarData/ModelarDB-RS/blob/master/server/src/storage/uncompressed_data_manager.rs#L56) and [`ErrorBound::new()`](https://github.com/ModelarData/ModelarDB-RS/blob/master/server/src/models/mod.rs#L63).

Reply:
I think the current comment seems fine.

Reply:
I agree, let's evaluate at a later time.